test/unit/SandboxCoordinatorSpec.ts

```diff
@@ -28,15 +28,30 @@ describe('SandboxCoordinator', () => {
   let sinonSandbox: sinon.SinonSandbox;
   let firstSandbox: { initialize: sinon.SinonStub, run: sinon.SinonStub, runMutant: sinon.SinonStub, dispose: sinon.SinonStub };
   let secondSandbox: { initialize: sinon.SinonStub, run: sinon.SinonStub, runMutant: sinon.SinonStub, dispose: sinon.SinonStub };
-  let options: StrykerOptions;
+  let options: Config;
   let reporter: any;
   let coverageInstrumenter: any;
   const expectedTestFramework: any = 'expected test framework';
   const expectedRunResult = { isTheExpectedRunResult: true };
   const expectedInputFiles: any = { isInputFiles: true };
 
   beforeEach(() => {
-    options = {};
+    options = {    
```

Comment:
This adds a lot of incidental detail (detail that is not needed for the test). I think its acceptable to add an `any` cast here:

```typescript
    options = <any>{};
```

What you can also do is add a helper method to the `helpers/producers.ts` file where you initialize a full config.